app/src/geo/google.js

```diff
@@ -21,43 +65,27 @@ module.exports = {
     },
 
     getAddress: function(location, callback) {
-        geocoder.reverse(location, function(err, geocodeResult) {
-            if (err) log.error(err);
+        var cacheKey = location.lat + "-" + location.lon;
```

Comment:
don't use string + concat like this please 

use 
let cacheKey = `${location.lat}-${location.lon}`;

instead